### DISCUSSION TITLE
Integration with Apollo Client

### DISCUSSION THREAD

**Discussion by nareshbhatia** [Urigo/graphql-scalars#957] [Q&A] [answered]
How do I integrate graphql-scalars with Apollo Client. I am using the `DateTime` scalar and it is not being deserialized as a `Date` object - coming through as a string.

**Comment by ardatan** [comment, marked as answer]
This is something related to Apollo Client;
See https://github.com/apollographql/apollo-client/issues/585

**Comment by nareshbhatia** [comment]
Thanks for the reference @ardatan. I read the full thread + the follow on thread (https://github.com/apollographql/apollo-feature-requests/issues/2). It is baffling that there is no easy solution for this after 4 years of discussion!

1. Curious if graphql-scalars is usable on any other GraphQL client (other than Apollo Client) or maybe with [graphql-codegen](https://www.graphql-code-generator.com/).
2. I see that the documentation describes integration with Apollo Server. So is there some workaround that people use on the client side? Otherwise, just doing this stuff on the server-side seems like wasted effort. Am I missing something?

**Reply by ardatan** [reply]
This one maybe?
https://github.com/eturino/apollo-link-scalars

**Reply by nareshbhatia** [reply]
Looks promising. Will take a look. Thanks, @ardatan.
